iMichaelOwolabi/my-diarium-backend
# 167608552 Setup project repository

#### What does this PR do?
Setup the project repository with the initial basic boilerplate
#### Description of Task to be completed?
* Configure Babel
* Configure ESLint
* Integrate Circle Ci
#### How should this be manually tested?
Clone the project repo and cd into the directory
- run npm  install to install all dependencies
- run npm run dev
- You should see the project server running
#### Any background context you want to provide?
This task is just to setup the project repo to take out redundant task out of the way
#### What are the relevant pivotal tracker stories?
[#167608552](https://www.pivotaltracker.com/story/show/167608552)
[#167622083](https://www.pivotaltracker.com/story/show/167622083)
